src/components/AccountForms/LoginForm/index.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { API_PATH } from 'utils/constants';
+import request from 'utils/axiosConfig';
+import { useAuth } from '../../../context/auth';
+import { GlobalContext } from 'context/global';
+
+import { Link, useHistory } from 'react-router-dom';
+import useStyles from './LoginForm.styles';
+import {
+  Paper,
+  TextField,
+  InputAdornment,
+  IconButton,
+  Button,
+} from '@material-ui/core';
+import { Visibility, VisibilityOff, Error } from '@material-ui/icons';
+import { ReactComponent as LoginHero } from './assets/LoginHero.svg';
+import { ReactComponent as TwitterLogo } from './assets/TwitterLogo.svg';
+
+const FORM_TITLE = 'Welcome back!';
+
+const emailRequirements = {
+  required: true,
+  pattern: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+};
+
+const LoginForm = ({ testOnSubmit }) => {
+  const [showPassword, setShowPassword] = useState(false);
+  const { register, handleSubmit, errors } = useForm();
+  const {
+    globalState: { isMobile },
+  } = useContext(GlobalContext);
+  const history = useHistory();
+  const classes = useStyles({
+    isMobile: isMobile,
+  });
+
+  const { auth } = useAuth();
+
+  useEffect(() => {
+    if (auth.isLoggedIn()) {
+      history.replace('/mentors');
+    }
+  }, [auth, history]);
+
+  const onSubmit = (credentials) => {
+    request.post(API_PATH.LOGIN, credentials).then((response) => {
+      if (response.status === 200) {
+        auth.checkCookie();
+        history.push('/mentors');
+      }
+    });
+  };
+
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const handleClickRegister = () => {
+    history.push('/register');
+  };
+
+  const content = (
+    <div className={classes.root}>
+      <Paper classes={{ root: classes.paperContainer }}>
+        <LoginHero className={classes.heroImage} />
+        <div className={classes.loginFormContainer}>
+          {(errors.email || errors.password) && (
+            <div className={classes.error} data-testid="authError">
+              <Error />
+              <p role="alert">Sorry, invalid email or password. Try again?</p>
+            </div>
+          )}
```

Comment:
### Changes:
we also want the error to be driven by query params and show the session timeout if the url has timedOut=true, for example https://femmecubator.com/login?timedOut=true

**Timed Out Session**
![image](https://user-images.githubusercontent.com/42846522/101265942-e4111680-3718-11eb-9f7c-33b866f296b8.png)